### PR TITLE
Implement std::cmp::PartialOrd for OwnedValue

### PR DESCRIPTION
**Pull request by benclmnt**
Related to #123

### CODE REVIEW

File: core/vdbe.rs [hunk 1]
```diff
@@ -704,6 +755,23 @@ impl Program {
...
+                        }
+                        (OwnedValue::Float(lhs), OwnedValue::Integer(rhs)) => {
+                            if *lhs <= (*rhs as f64) {
+                                state.pc = target_pc;
+                            } else {
+                                state.pc += 1;
+                            }
+                        }
+                        (_, OwnedValue::Null) | (OwnedValue::Null, _) => {
+                            state.pc = target_pc;
```
Comment (jussisaurio) [inline core/vdbe.rs:773]: I guess these instructions should also have a register for `SQLITE_JUMPIFNULL`, but probably not a thing to concentrate on right now

**Review by jussisaurio — APPROVED** [review]

**Comment by penberg** [comment]
Looks pretty straight-forward. @pereman2, are you OK with this as well?

File: core/vdbe.rs [hunk 2]
```diff
@@ -675,6 +709,23 @@ impl Program {
                                 state.pc += 1;
                             }
                         }
+                        (OwnedValue::Integer(lhs), OwnedValue::Float(rhs)) => {
```
Comment (pereman2) [inline core/vdbe.rs:712]: Are these operations needed? We can just overlod operator == like with sums.

**Review by pereman2 — CHANGES_REQUESTED** [review]
Sorry but i implement https://github.com/penberg/limbo/pull/152. The rest maybe we can keep

**Comment by penberg** [comment]
Hey @benclmnt, sorry for the hopefully trivial conflict in `all.test`. @pereman2, please re-review this.

**Comment by benclmnt** [comment]
> Hey @benclmnt, sorry for the hopefully trivial conflict in `all.test`. @pereman2, please re-review this.

no worries, fixed it

**Review by pereman2 — APPROVED** [review]

**Pull request merged**